docs/book/src/reference/providers.md

```diff
@@ -12,9 +12,9 @@ support v1alpha2 API types.
 
 ## Infrastructure
 - [**AWS**](https://github.com/kubernetes-sigs/cluster-api-provider-aws)
-- [Azure](https://github.com/kubernetes-sigs/cluster-api-provider-azure)
+- [**Azure**](https://github.com/kubernetes-sigs/cluster-api-provider-azure)
 - [Baidu Cloud](https://github.com/baidu/cluster-api-provider-baiducloud)
-- [Metal3](https://github.com/metal3-io/cluster-api-provider-metal3)
+- [**Metal3**](https://github.com/metal3-io/cluster-api-provider-metal3)
 - [DigitalOcean](https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean)
 - [Exoscale](https://github.com/exoscale/cluster-api-provider-exoscale)
 - [GCP](https://github.com/kubernetes-sigs/cluster-api-provider-gcp)
```

Comment:
does GCP support v1alpha3? didn't see a release for it

Reply:
[it appears to](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/master/examples/cluster/cluster.yaml#L2), but i would leave it to @vincepri to decide if it makes sense to highlight it here.

Reply:
It hasn’t been updated in a while, not sure about the state of things today

Reply:
I'm still working on getting the latest changes needed for v1alpha3 support in place.